src/components/Board.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { determineWin } from '../utils/determineWin'
+import { Moves } from './Moves'
+import { Square } from './Square'
+import { Status } from './Status'
+
+export function Board(): JSX.Element {
+  const [turn, setTurn] = useState(0)
+  const [boardState, setBoardState] = useState([
+    {
+      board: ['', '', '', '', '', '', '', '', ''],
+    },
+  ])
+  const [playerOne, setPlayerOne] = useState(true)
+
+  /*
+    access boardState to record player's moves and save them into array;
+    this accesses the board property within boardState for each turn
+  */
+  const currentBoardState = boardState.slice(0, turn + 1)
+  const currentMove = currentBoardState[currentBoardState.length - 1]
+  const board = currentMove.board.slice(0)
+  const winner = determineWin(board)
```

Comment:
I think we can cut all of these variables here except `board`. `winner` is only used by the `Status` component which already gets the whole board. It can just call `determineWin(board)` internally to determine the winner. No need have a `winner` prop. 
```suggestion
    const board = [ ...boardState[boardState.length - 1].board ] // or boardState[boardState.length - 1].board.slice(0) if you like
```